srcpkgs/python3-jmespath/template

```diff
@@ -1,16 +1,17 @@
 # Template file for 'python3-jmespath'
 pkgname=python3-jmespath
-version=0.10.0
-revision=5
+version=1.0.1
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3"
+checkdepends="python3-pytest python3-hypothesis"
 short_desc="JSON Query language for Python3"
 maintainer="yopito <pierre.bourgin@free.fr>"
 license="MIT"
 homepage="http://jmespath.org"
```

Comment:
```suggestion
homepage="http://jmespath.org"
changelog="https://github.com/jmespath/jmespath.py/raw/develop/CHANGELOG.rst"
```

Reply:
fixed in push force below, thanks for the review